src/components/list/list-item.types.ts

```diff
@@ -49,6 +49,8 @@ export interface ListItem<T = any> {
 
 export interface ListSeparator {
     separator: true;
+
+    separatorTitle?: string;
```

Comment:
It might make sense to just name this `text` as for regular `ListItem`s

Reply:
It should at least not have `separator` in the name, we already know that it's a separator so we do not need to state that again :slightly_smiling_face:

Reply:
⚡